nx/lib/nx.ex

```diff
@@ -9167,6 +9167,84 @@ defmodule Nx do
     end
   end
 
+  @doc """
+  Finds the variance of a tensor.
+
+  ## Examples
+
+  iex> Nx.var(Nx.tensor([[1, 2], [3, 4]]))
+
+  #Nx.Tensor<
+  f32
+  1.25
+  >
+  """
+  @doc type: :element
+  @spec var(tensor :: Nx.Tensor.t()) :: Nx.Tensor.t()
+  def var(%Nx.Tensor{shape: shape} = tensor) do
```

Comment:
What is the definition for the variance of a 0-dimensional tensor? Perhaps we should have a check against that

Reply:
```
iex(3)> Nx.variance Nx.tensor(5)
#Nx.Tensor<
  f32
  0.0
>
```